src/components/NormalLeftRow/index.jsx

```diff
@@ -130,15 +130,18 @@ function NormalLeftRow({ classes, treeNode, refType, setSchemaTree }) {
     const specifications = Object.keys(schemaInput).filter(
```

Comment:
Would it be complicated to reduce the spacing between the title and description to 8px rather than 28px?

#### Before
![Screen Shot 2020-02-10 at 10 05 03 AM](https://user-images.githubusercontent.com/3766511/74161517-0f0c0600-4bed-11ea-9470-8a4de9e0d27e.png)

#### After
![Screen Shot 2020-02-10 at 10 04 56 AM](https://user-images.githubusercontent.com/3766511/74161518-0f0c0600-4bed-11ea-9cb8-dd57edc652db.png)

Reply:
If it's okay, I think maybe reducing the line height for descriptor lines (`title`, `description`) could be a solution. Since we define the height properties to provide consistency between lines and rows across the panels, there are no margin values we can access to alter the spacing between lines. So instead, maybe we can use smaller line heights just for descriptor keywords to reduce the spacing between the title and description. This way, we can maintain the row and line heights while also reducing the spacing just between the title and description :)

I've applied changes in 

![image](https://user-images.githubusercontent.com/29671309/74218611-d738b780-4ced-11ea-8d42-adff03b5059e.png)

Reply:
Since the lines the height property instead of using margin or padding properties in order to maintain consistency within the schemaTable, I've modified the title line to have a shorter height in cases where it is followed by a description line. This way, it has the same effect of or reducing the spacing while also making sure blank lines aren't shortened.
I've implemented the changes in f6566e5 and the result looks like this:

(left) **before**    ||     (right) **after**
![image](https://user-images.githubusercontent.com/29671309/74239574-745e1500-4d1b-11ea-92fd-4fec95454a2b.png)

Reply:
Since the lines use the height property instead of the margin or padding properties in order to maintain consistency within the schemaTable, we could modify the title line to have a shorter height in cases where it is followed by a description line. This way, we can reduce the spacing while avoiding problems such as reducing blank lines (which occur when using margin or paddings)

The result would look like this:

(left) **before**    ||     (right) **after**
![image](https://user-images.githubusercontent.com/29671309/74239574-745e1500-4d1b-11ea-92fd-4fec95454a2b.png)

However, one complication would be that we would also have to reduce the height of the left row's line for cases where the above apply. For example, in cases where there are no chips and title and description exist, we would have to shorten the left row's line as well to align the lines like this:

![image](https://user-images.githubusercontent.com/29671309/74242694-b7bb8200-4d21-11ea-9c47-219eba5525f5.png)

Reply:
Hm, maybe we should just keep the spacing for now and worry about this later. We can create an issue to keep track of this.

Reply:
Sure! Opened issue #73